Sources/OpenCombine/Publishers/Publishers.Throttle.swift

```diff
@@ -104,16 +104,6 @@ extension Publishers {
         }
 
         // swiftlint:disable generic_type_name
-
-        /// Attaches the specified subscriber to this publisher.
-        ///
-        /// Implementations of ``Publisher`` must implement this method.
-        ///
-        /// The provided implementation of ``Publisher/subscribe(_:)-4u8kn``calls
```

Comment:
Remove this as it is duplicated by the protocol's doc.

``Publisher/subscribe(_:)-4u8kn`` does not exist on OpenCombine. And we need to use ``Publisher/subscribe(_:)-199o9`` here. Otherwise we'll get a documentation generation warning here. And the link won't be resolved successfully.

Already fixed it on the doc of ``Publisher``.